.circleci/config.yml

```
@@ -46,6 +46,10 @@ commands:
             REDIS_PASSWORD="$REDIS_PASSWORD"
             REDIS_URL="$REDIS_URL"
             APP_IMAGE_NAME="$APP_IMAGE_NAME"
+            AWS_S3_BUCKET="$AWS_S3_BUCKET"
+            AWS_S3_ACCESS_KEY="$AWS_S3_ACCESS_KEY"
+            AWS_S3_KEY_SECRET="$AWS_S3_KEY_SECRET"
```

```suggestion
            AWS_ACCESS_KEY_ID="$AWS_ACCESS_KEY_ID"
            AWS_SECRET_ACCESS_KEY ="$AWS_SECRET_ACCESS_KEY"
```
you should use names from the documentation
https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/guide_credentials_environment.html

---

I thought I already commented on this in a previous PR. Did you miss that comment?

---

Yep, I totally missed that by accident 😬

---

Ohh, I see. I only changed `example.env` file.